src/assessments/assessment-builder.tsx

```diff
@@ -87,21 +87,21 @@ export class AssessmentBuilder {
         return requirementLink.renderRequirementDescriptionWithIndex();
     }
 
-    private static enableTest(scanData: IScanData, payload: AssessmentToggleActionPayload): void {
-        const scanAssessmentData = scanData as IAssessmentScanData;
+    private static enableTest(scanData: ScanData, payload: AssessmentToggleActionPayload): void {
+        const scanAssessmentData = scanData as AssessmentScanData;
         scanAssessmentData.enabled = true;
         scanAssessmentData.stepStatus[payload.requirement] = true;
     }
 
-    private static disableTest(scanData: IScanData, requirement: string): void {
-        const scanAssessmentData = scanData as IAssessmentScanData;
-        scanAssessmentData.stepStatus[requirement] = false;
+    private static disableTest(scanData: ScanData, step: string): void {
```

Comment:
This looks like it's reverting an earlier change to rename step to requirement. Bad merge?

Reply:
Bad merge indeed. Thanks for noticing.

Reply:
@dbjorge Updated